content/go/CDN_services.go.md

```diff
@@ -0,0 +1,568 @@
+---
+title: "CDN_services.go"
+description: "CDN_services.go"
```

Comment:
Change the description to "How to interact with the CdnMarketplace services. Order CDN, manage mappings and other related functions"